tests/pio_tests/engines/recommendation-engine/build.sbt

```diff
@@ -26,6 +26,6 @@ name := "template-scala-parallel-recommendation"
 organization := "org.apache.predictionio"
 
 libraryDependencies ++= Seq(
-  "org.apache.predictionio" %% "apache-predictionio-core" % "0.14.0" % "provided",
+  "org.apache.predictionio" % "apache-predictionio-core" % "0.15.0-SNAPSHOT" % "provided",
```

Comment:
We need to use the current (local) SNAPSHOT here, otherwise things like 2.12 support will never be able to pass in Travis before being merged.

Reply:
@shimamoto Do you have any idea why this fails to resolve? It should have been published locally inside docker on make-distribution, no? We need to be testing the current snapshot here.

Reply:
@EmergentOrder You need to write the double `%%` after the groupID.

Reply:
Ah of course. I removed that and then missed putting it back. Thanks.